Documentation/ceph-upgrade.md

```diff
@@ -115,21 +114,6 @@ In order to successfully upgrade a Rook cluster, the following prerequisites mus
 * All pods consuming Rook storage should be created, running, and in a steady state. No Rook
   persistent volumes should be in the act of being created or deleted.
 
-### Helm
```

Comment:
This section doesn't seem necessary anymore, it has already been documented since v1.3